djstripe/models.py

```diff
@@ -52,6 +51,40 @@
 djstripe_settings.set_stripe_api_version()
 
 
+class PaymentMethod(models.Model):
+    """
+    An internal model that abstracts the legacy Card and BankAccount
+    objects with Source objects.
+
+    Contains two fields: `id` and `type`:
+    - `id` is the id of the Stripe object.
+    - `type` can be `card`, `bank_account` or `source`.
+    """
+    id = CharField(max_length=255, primary_key=True)
+    type = CharField(max_length=12, db_index=True)
```

Comment:
There's no method here to enumerate payment sources for a customer (it has to be done via Card atm).